code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -443,3 +448,14 @@ GLOBAL_LIST_INIT(strain_list, init_glob_strain_list())
 	COOLDOWN_DECLARE(xeno_resting_cooldown)
 	///The unresting cooldown
 	COOLDOWN_DECLARE(xeno_unresting_cooldown)
+
+/mob/living/carbon/xenomorph/proc/lifesteal(attacker, target) //For globadiers lifesteal debuff
```

Comment:
Please name it something more generic since you're registering it to such a generic signal. 

Also docs